faq/serverless-containers.mdx

```diff
@@ -75,6 +75,23 @@ Serverless Containers are billed on a pay-as-you-go basis, strictly on resource
 
 **Monthly Cost: €22.35**
 
+## How to select the right resources (vCPU/RAM) for Serverless Containers ?
+
+Too low vCPU and RAM values can lead to containers going to error status so it's important to select enough of vCPU and RAM.
+The recommended approach is to set high values and then use metrics [(see documentation)](/serverless/containers/how-to/monitor-container/) to monitor
+the usage of resources and to adjust the value.
+
```

Comment:
```suggestion
If vCPU and RAM values are too low, it can lead to containers going into error status. It is important to select the correct amount of vCPU and RAM.
The recommended approach is to set high values and then use metrics [(see documentation)](/serverless/containers/how-to/monitor-container/) to monitor
the usage of resources and to adjust the value.
```